stripe.go

```diff
@@ -450,19 +418,15 @@ func (s *BackendImplementation) Do(req *http.Request, body *bytes.Buffer, v inte
 
 	resBody, err := ioutil.ReadAll(res.Body)
 	if err != nil {
-		if s.LogLevel > 0 {
-			s.Logger.Printf("Cannot read response: %v\n", err)
-		}
+		s.LeveledLogger.Errorf("Cannot read response: %v", err)
 		return err
 	}
 
 	if res.StatusCode >= 400 {
 		return s.ResponseToError(res, resBody)
 	}
 
-	if s.LogLevel > 2 {
-		s.Logger.Printf("Response: %s\n", string(resBody))
-	}
+	s.LeveledLogger.Errorf("Response: %s\n", string(resBody))
```

Comment:
Should this be `Infof`?

Reply:
I was mostly going from the fact that it was `s.LogLevel > 2` before

Reply:
Err, yes .... :)

I'd say even `Debugf` is more appropriate even given that this is raw data. Going to change to that.